runtime/oti/j9.h

```diff
@@ -348,7 +348,7 @@ static const struct { \
 		J9_IS_J9CLASS_FLATTENED(fieldClazz) && \
 		(J9_ARE_NO_BITS_SET((romFieldShape)->modifiers, J9AccVolatile) || (J9CLASS_UNPADDED_INSTANCE_SIZE(fieldClazz) <= sizeof(U_64))))
 #define J9_VALUETYPE_FLATTENED_SIZE(clazz) (J9CLASS_HAS_4BYTE_PREPADDING((clazz)) ? ((clazz)->totalInstanceSize - sizeof(U_32)) : (clazz)->totalInstanceSize)
-#define IS_REF_OR_VAL_SIGNATURE(firstChar) ('L' == (firstChar) || 'Q' == (firstChar))
+#define IS_REF_OR_VAL_SIGNATURE(firstChar) ('L' == (firstChar))
```

Comment:
Since this line and line 360 are now the same. We can move this macro outside of `#if defined(J9VM_OPT_VALHALLA_FLATTENABLE_VALUE_TYPES)`

Reply:
Fixed.